app/models/email_message.rb

```diff
@@ -1,4 +1,6 @@
 class EmailMessage < Ahoy::Message
+  belongs_to :feedback_message, optional: true
```

Comment:
If I understood the code correctly an email message can optionally only belong to a feedback message